src/components/spaces/show/index.js

```diff
@@ -330,7 +330,7 @@ export default class SpacesShow extends Component {
                 existingVariableNames={organizationFacts.map(e.facts.getVar)}
                 facts={organizationFacts}
                 organization={organization}
-                isEditable={false}
+                isEditable={true}
```

Comment:
This doesn't actually seem to prevent editing of the individual facts BTW. No big deal for now, though.